packages/sources/css/src/components/text-input/src/index.css

```diff
@@ -86,3 +106,9 @@ textarea.vtmn-text-input {
 .vtmn-text-input--error {
   box-shadow: inset 0 0 0 rem(2px) var(--vtmn-color_danger);
 }
+
+.vtmn-text-input--valid + span {
+  position: relative;
+  right: rem(32px);
+  top: rem(12px);
+}
```

Comment:
You can move it just after `vtmn-text-input--valid` declaration please?

Reply:
Done, thanks !